app.rb

```diff
@@ -0,0 +1,108 @@
+require_relative 'book'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'rental'
+
+class App
+  attr_accessor :book_lists, :user_lists, :rental_lists
+
+  def initialize
+    @book_lists = []
+    @user_lists = []
+    @rental_lists = []
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+    case person_type
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid option'
+    end
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    puts 'Has parent permission? [y/n]'
+    parent_permission = gets.chomp
+    parent_permission = case parent_permission.downcase
+                        when 'y'
+                          true
+                        else
+                          false
+                        end
+    puts 'Classroom:'
+    classroom = gets.chomp
+    student = Student.new(classroom, age, name, parent_permission)
+    @user_lists.push(student)
+    puts 'Person created successfully'
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    teacher = Teacher.new(specialization, age, name, parent_permission: true)
+    @user_lists.push(teacher)
+    puts 'Person created successfully'
+  end
```

Comment:
The create_teacher function throws the error below. Kindly fix it.

![image](https://user-images.githubusercontent.com/27709832/233619022-25670023-fd41-4698-b34b-453f9843e9b3.png)